project1/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cranes wife</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Abel|IM+Fell+French+Canon+SC|Notable&display=swap"
+        rel="stylesheet">
+
+</head>
+
+<body>
+    <hr />
+    <h1>Crane Wife</h1>
+    <hr />
+    <h2><a href="https://cjhauser.com/>"><em>By CJ Hauser</em></a></h2>
+
+    <div class=sub-border>
+        <p class="date"> July 16, 2019</p>
+        <p> Ten days after I called off my engagement I was supposed to go on a scientific expedition to study the
+            whooping
+            crane on the gulf coast of Texas. Surely, I will cancel this trip, I thought, as I shopped for nylon hiking
+            pants
+            that zipped off at the knee. Surely, a person who calls off a wedding is meant to be sitting sadly at home,
+            reflecting on the enormity of what has transpired and not doing whatever it is I am about to be doing that
+            requires
+            a pair of plastic clogs with drainage holes. Surely, I thought, as I tried on a very large and floppy hat
+            featuring
+            a pull cord that fastened beneath my chin, it would be wrong to even be wearing a hat that looks like this
+            when
+            something in my life has gone so terribly wrong.</p>
+
+        <p> Ten days earlier I had cried and I had yelled and I had packed up my dog and driven away from the upstate
+            New
+            York
+            house with two willow trees I had bought with my fiancé.</p>
+
+        <p> Ten days later and I didn’t want to do anything I was supposed to do.</p>
+
+        <div class=star>*</div>
```

Comment:
Classname should be wrapped in double quotes:
```suggestion
        <div class="star">*</div>
```